app/config/factionhall/rooms.json

```diff
@@ -113,17 +114,17 @@
         },
         {
             "id": "large-lounge",
-            "name": "Comet Pizza Parlor",
+            "name": "Drawing Room",
             "type": "CHATSTREAM",
             "capacity": 666,
             "doorTint": "#000000",
             "muteRoom": true,
             "hideVideo": true,
-            "backgroundImage": "https://s3.us-east-2.amazonaws.com/thesatanic.estate/backgrounds/factionhall/greyfaction/gfcometpizzaparlor.png",
+            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/Sober+Faction+2022/SF+S3.jpeg",
             "iframeOptions": {
-                "src": "https://www.youtube.com/embed/videoseries?list=PLVo0KZZdkRu0Dr0UjZLwHMSTbfidYdazc"
+                "src": "https://www.youtube.com/embed/sTNQBOBxsxU"
             },
-            "description": "Let's go check out the basement!",
+            "description": "Dr. Lucy Hall - In 1888, served as the physician in charge of the Reformatory Prison in Sherborn, Massachusetts. She studied 204 “inebriate” women in her care, and found that more than one-third of the married women had been so beaten by their drunken husbands that they had scars on their heads.",
```

Comment:
This is going to break the PR. Remove the quotes around inebriate and replace with apostrophes.